frame/balances/src/tests_composite.rs

```diff
@@ -97,7 +97,7 @@ impl Config for Test {
 	type DustRemoval = ();
 	type Event = Event;
 	type ExistentialDeposit = ExistentialDeposit;
-	type AccountStore = frame_system::Pallet<Test>;
+	type AccountStore = system::Pallet<Test>;
```

Comment:
this change is probably a miss in a merge no ?
```suggestion
	type AccountStore = frame_system::Pallet<Test>;
```